docs/canonicalk8s/charm/howto/custom-registry.md

```diff
@@ -88,6 +88,24 @@ juju config k8s containerd-custom-registries='[{
 }]'
 ```
 
+### Sharing with all workers in the cluster
+
+The Juju configuration only applies to control-plane nodes, but is likely
```

Comment:
I guess this "likely" here hints that you may want to have some workers not follow the configuration of the entire cluster?

Do you think we should eliminate this uncertainty and skip the first sentence entirely?